ansible/tasks/validation.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- debug:
```

Comment:
Maybe have a more useful message with this debug var, if we want to print it

Reply:
a leftover - we can drop it imho